src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -171,6 +171,8 @@
                             var text = requestStack.length + ' AJAX request' + (requestStack.length > 1 ? 's' : '');
                             infoSpan.textContent = text;
                         }
+
+                        ajaxToolbarPanel.style.display = 'block';
```

Comment:
I find this practice a bit odd ... but if others agree, I'll do the change. Thanks.